src/libMediator/Mediator.h

```diff
@@ -33,12 +33,52 @@
 #include "libNode/Node.h"
 #include "libValidator/Validator.h"
 
+/// System test control class
+class SysTestData {
+  bool TestStarted = false;  
+  bool IsANode = false;
+  uint64_t TestStartEpoch = 4;
+  uint64_t CurrentEpoch = 0;
+  uint16_t MyNodeId = 0;
+  uint16_t FirstAId = 0;
+
+ public:
+  /// Constructor
+  SysTestData() {
+    TestStartEpoch = TEST_START_EPOCH;
```

Comment:
Do you intend to change these values some time during operation? If not, we can just directly use the constants.